build.sbt

```diff
@@ -162,7 +166,8 @@ lazy val docs = Project(
     ),
     resolvers += Resolver.jcenterRepo,
     publishRsyncArtifact := makeSite.value -> "www/",
-    publishRsyncHost := "akkarepo@gustav.akka.io"
+    publishRsyncHost := "akkarepo@gustav.akka.io",
+    crossScalaVersions := List(scala212, scala213),
```

Comment:
docs need cross versions?

Reply:
I think I did this because docs depends on cross-built modules, and in that case sbt requires the 'depending' module to also be cross-built (https://github.com/sbt/sbt/issues/3465)